packages/web/pages/landing.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import Intro from '../components/Landing/Intro';
+import Game from '../components/Landing/Game';
+import Build from '../components/Landing/Build';
+import Revolution from '../components/Landing/Revolution';
+import WildWeb from '../components/Landing/WildWeb';
+import Frontier from '../components/Landing/Frontier';
+import Together from '../components/Landing/Together';
+import WhatWeDo from '../components/Landing/WhatWeDo';
+import Optimal from '../components/Landing/Optimal';
+import Unplug from '../components/Landing/Unplug';
+import Who from '../components/Landing/Who';
+import JustWatch from '../components/Landing/JustWatch';
+import Cards from '../components/Landing/Cards';
+
+function Landing(): JSX.Element {
+  return <div>
+    <Intro />
+    <Game />
+    <Build />
+    <Revolution/>
+    <WildWeb />
+    <Frontier />
+    <Together /> 
+    <WhatWeDo /> 
+    <Optimal />
+    <Unplug />
+    <Who />
+    <Cards /> 
+    <JustWatch />
+
+  </div>;
+}
+
+export default Landing;
```

Comment:
Sort the dependencies and hide the megamenu:

```suggestion
import React from 'react';

import Build from '../components/Landing/Build';
import Cards from '../components/Landing/Cards';
import Frontier from '../components/Landing/Frontier';
import Game from '../components/Landing/Game';
import Intro from '../components/Landing/Intro';
import JustWatch from '../components/Landing/JustWatch';
import Optimal from '../components/Landing/Optimal';
import Revolution from '../components/Landing/Revolution';
import Together from '../components/Landing/Together';
import Unplug from '../components/Landing/Unplug';
import WhatWeDo from '../components/Landing/WhatWeDo';
import Who from '../components/Landing/Who';
import WildWeb from '../components/Landing/WildWeb';

export const getStaticProps = async () => ({
  props: {
    hideTopMenu: true,
  },
});

const Landing: React.FC = () => (
  <div>
    <Intro />
    <Game />
    <Build />
    <Revolution />
    <WildWeb />
    <Frontier />
    <Together />
    <WhatWeDo />
    <Optimal />
    <Unplug />
    <Who />
    <Cards />
    <JustWatch />
  </div>
);

export default Landing;
```

Reply:
> Sort the dependencies and maybe hide the megamenu? @davort what do you think about hidning the megamenu on the landing page?

I was just about to mention this. 

I figured get the initial mockup done and then come back to this later once I got more info on this :)